portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +14,23 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+  List<String> comments = List.of("Learning a lot of new concepts!", "Everybody is very supportive!","Relearning Java right now");
+  Map<String,List<String>> commentsMap = new HashMap<String,List<String>>();
+  
   //GET request handler
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Cindy!</h1>");
+    commentsMap.put("comments",comments);
+    String json = convertToJsonUsingGson(commentsMap);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+
+  }
+  private String convertToJsonUsingGson(Map<String,List<String>> map) {
+    Gson gson = new Gson();
+    String json = gson.toJson(map);
+    return json;
   }
```

Comment:
You can leave this if you want for future use, but with above suggestions it would not be used anymore.